### ISSUE TITLE
Switch to u64/u128 usage for circuit size measurements

### ISSUE THREAD

**Issue by CPerezz** [dusk-network/plonk#51]
Before we can change from usize to u64, the permutation.rs file needs to be modified as the wire encoding relies on u32, probably better to leave this until second/third iteration

_Originally posted by @decentralisedkev in https://github.com/dusk-network/plonk/issues/37#issuecomment-572811268_

**Comment by decentralisedkev** [comment]
I think this is a fine compromise actually. Rust does not silently cast usize to u32/u64, so casting in the places that it is needed should be fine.

We use usize to do quite a bit of indexing, so in this regard, it might be better to leave as is and explicitly cast when necessary

**Comment by Bounce23** [comment]
Okay, then I'll rename this issue to make it relevant to a refactor where we do case necessity casting.

**Comment by decentralisedkev** [comment]
I believe we already do this, as Rust does not silently cast types

**Comment by decentralisedkev** [comment]
Clicked the wrong button. Nevertheless, I think this can be closed as it is no longer relevant
